articles/templating-your-static-site/index.md

```diff
@@ -0,0 +1,188 @@
+
+### Installing Eleventy
+
+The first step is to install Eleventy which will be done through the package manager, NPM. New to NPM? Check out my installation guide **(link to NPM install in Converting a Static Site to a Dynamic NodeJS Web App article)**.
+
+Create a package.json file using `npm init -y` and then install Eleventy using `npm install --save-dev @11ty/eleventy`.
+
+To test the install, run `npx @11ty/eleventy` which should return:
+`Processed X files in number of seconds (version number)`.
+
+### Serving Your Static Site using Eleventy
+
+The second step is to be able to serve your current site using Eleventy.
+
+To do so, you'll have to create a config file and add some settings to it which will tell Eleventy how to handle your images, CSS and JavaScript.
+
+In the root folder of your site, create a JS file called .eleventy.js and include the following:
+
+```js
+module.exports = function (eleventyConfig) {
+    eleventyConfig.addPassthroughCopy('img');
+    eleventyConfig.addPassthroughCopy('css');
+    eleventyConfig.addPassthroughCopy('js');
+    return {
+        passthroughFileCopy: true
+    };
+};
+```
+This will tell Eleventy when it builds your website to just copy the img, CSS and js folders across (which is called passthrough copy) so you can link them to them in your code.
+
+Now run `npx @11ty/eleventy` and it should have built your website and output the result to the `_site` folder. Congratulations, you've just finished the first Eleventy build of your website.
+
+**Tip:** If you're using the version control software, Git **link to a section article on Git** manages your code then it would be a good idea to add the `_site` folder to your .gitignore file. This will stop you from committing your `_site` folder to Git which prevents the constant uncommitted changes warning.
+
+### Creating Your First EJS Partial
+
+Now that your site is up and running on Eleventy, we can get to the templating part.
+
+EJS partials are a way of reusing pieces of code for example, if you created an EJS partial of your website's header then it could be referenced on all your website's pages. In the future, if you decided to add a new menu item then you would only need to update one file.
+
+Create a folder in the root of your website called `_includes`. This is where Eleventy looks for templates.
+
+**Tip:** A naming convention you may wish to use is to add an `_` to your template files so `_head.ejs`.
+
+In your `_includes` folder, add a file called `_head.ejs` and copy and paste the content of your website's head section. Don't worry if the content differs between pages such as the title tag. We will cover how to use EJS variables with Eleventy in the Adding More Front-Matter section **Anchor Link** so such values can be unique for each page.
+
+Add new partial files for other repeated elements such as the header and footer.
```

Comment:
Consider adding a screenshot as an example.

Reply:
Added screenshot of folder structure and added example `_head.ejs` file code block.